Compiler Warning

```
In file included from sicxe_asm.cpp:18:0:
sicxe_asm.h: In function 'std::ofstream& operator<<(std::ofstream&, const file_parser::formatted_line&)':
sicxe_asm.h:146:9: warning: unused variable 'machine_code' [-Wunused-variable]
     int machine_code = f_l.machinecode;
         ^
```

--- a/sicxe_asm.h
+++ b/sicxe_asm.h
@@ -147,7 +147,6 @@ inline std::ofstream &operator<<(std::ofstream &out, const file_parser::formatte
     const int label_col_width = 10;
     const int opcode_col_width = 11;
     const int operand_col_width = 12;
-    int machine_code = f_l.machinecode;
 
     out << std::setw(linenum_col_width) << std::right << std::setfill(' ') << f_l.linenum << "     ";
     out << std::setw(address_col_width) << std::right << std::setfill(' ') << f_l.address << "     ";

